File: JavaScript projects/JavaScript Basics/Conditional Statements/exercises/05_godzilla_vs_kong.js
function godzillaVsKong(budget, countBackgroundActors, priceEquipment){
    let priceScenery = budget * 0.1;
    let totalPriceEquipment = countBackgroundActors * priceEquipment;
    if (150 < countBackgroundActors)
        totalPriceEquipment *= 0.9;
    let totalSumCosts = priceScenery + totalPriceEquipment;
    if (totalSumCosts <= budget){
        console.log(`Action!`);
        console.log(`Wingard starts filming with ${(budget - totalSumCosts).toFixed(2)} leva left.`);
        }else{
            console.log(`Not enough money!`);
            console.log(`Wingard needs ${(totalSumCosts - budget).toFixed(2)} leva more.`);
        }
}
godzillaVsKong(9587.88,222,55.68);

File: Python projects/Python Basics/02_conditional_statements/exercises/05_godzilla_vs_kong.py
budget = float(input())
actors = int(input())
price_clothes = float(input())
decor = budget * 0.1
if actors > 150:
    cost_cloths = price_clothes * actors * 0.9
else:
    cost_cloths = price_clothes * actors
movie_cost = decor + cost_cloths
if movie_cost > budget:
    print(f"Not enough money!\nWingard needs {movie_cost - budget:.2f} leva more.")
else:
    print(f"Action!\nWingard starts filming with {budget - movie_cost:.2f} leva left.")
